api/v1alpha/framework/xcuitest.schema.json

```diff
@@ -247,7 +247,8 @@
             "description": "When shard is configured as concurrency, saucectl automatically splits the tests by concurrency so that they can easily run in parallel.",
```

Comment:
Should this description be updated to something more general, now that there's more than one sharding option? 🤔

Reply:
Yah, good call.